v3/v3_structs.go

```diff
@@ -2356,14 +2356,27 @@ type ProtectionRuleInput struct {
 
 // RecoveryPlanResources represents the resources of recovery plan
 type RecoveryPlanResources struct {
-	StageList  []*StageList `json:"stage_list,omitempty"`
-	Parameters *Parameters  `json:"parameters,omitempty"`
+	VolumeGroupRecoveryInfoList []*VolumeGroupRecoveryInfo `json:"volume_group_recovery_info_list,omitempty"`
+	StageList                   []*StageList               `json:"stage_list,omitempty"`
+	Parameters                  *Parameters                `json:"parameters,omitempty"`
+}
+
+// VolumeGroupRecoveryInfo encapsulates the specification of volume
+// groups to be included in the recovery plan.
+type VolumeGroupRecoveryInfo struct {
+	CategoryFilter *CategoryFilter `json:"category_filter,omitempty"`
 }
 
 // Parameters represents a object for resource of recovery plan
 type Parameters struct {
-	FloatingIPAssignmentList []*FloatingIPAssignmentList `json:"floating_ip_assignment_list,omitempty"`
-	NetworkMappingList       []*NetworkMappingList       `json:"network_mapping_list,omitempty"`
+	FloatingIPAssignmentList []*FloatingIPAssignmentList    `json:"floating_ip_assignment_list,omitempty"`
+	NetworkMappingList       []*NetworkMappingList          `json:"network_mapping_list,omitempty"`
+	AvailabilityZoneList     []*AvailabilityZoneInformation `json:"availability_zone_list,omitempty"`
+}
+
+type AvailabilityZoneInformation struct {
```

Comment:
Please add a comment for all exported types.

Reply:
@thunderboltsid I've addressed your comments. Please take a look.